.github/workflows/linting.yaml

```
@@ -29,3 +29,18 @@ jobs:
       - run: npm ci
 
       - run: npm run lint:eslint:check
+    commitlint:
```

actionlint (1.7.12)

[error] unexpected key "commitlint" for "job" section. expected one of "concurrency", "container", "continue-on-error", "defaults", "env", "environment", "if", "name", "needs", "outputs", "permissions", "runs-on", "secrets", "services", "snapshot", "steps", "strategy", "timeout-minutes", "uses", "with"

(syntax-check)